code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -286,6 +286,8 @@
 
 	if(isobj(crushed))
 		var/obj/crushed_obj = crushed
+		if(istype(crushed_obj, /obj/structure/resin/silo) || istype(crushed_obj, /obj/structure/resin/xeno_turret))
```

Comment:
maybe a flag would be cleaner? Like CRUSHER_CHARGE_IMMUNE or something

Reply:
Perhaps. I am hoping to maybe just apply it to all resin structures in the future if the need to smash resin doors permanently goes away.